server/container/guice/cassandra-guice/README.adoc

```diff
@@ -5,15 +5,21 @@
 === Requirements
 
  * Java 8 SDK
- * Docker
- * Maven
+ * Docker or ElasticSearch 2.2.1 and Cassandra 2.2.3
+ * Maven 3
 
 === Building the artifacts
 
 An usual compilation using maven will produce two artifacts into target directory :
 
- * james-server-cassandra-guice-${version}.jar
- * james-server-cassandra-guice-${version}.lib
+ * james-server-cassandra-guice.jar
+ * james-server-cassandra-guice.lib
+
+You can for example run in the base of this git repository :
+
+```
+mvn clean install -Dskiptests
```

Comment:
Nope, profiles only enable MPT tests / JMAP integration tests / webadmin tests. To produce the jars, this is enougth